packages/calcite-components/src/controllers/useFocusTrap.ts

```diff
@@ -53,6 +53,38 @@ interface FocusTrapComponent extends LitElement {
    * When defined, provides a condition to disable focus trapping. When `true`, prevents focus trapping.
    */
   focusTrapDisabledOverride?: () => boolean;
+
+  /**
+   * Additional options to configure the focus trap.
+   */
+  focusTrapOptions?: ExtendedFocusTrapOptions;
+}
+
+export type ExtendedFocusTrapOptions =
+  /**
+   * @see https://github.com/focus-trap/focus-trap#createoptions
+   */
+  Pick<FocusTrapOptions, "allowOutsideClick" | "initialFocus" | "returnFocusOnDeactivate"> & {
+    /**
```

Comment:
How come this doesn't include `escapeDeactivates`?

Reply:
I didn't include some options that may conflict with existing component props (e.g., `escapeDisabled), but we could revisit this if there's a use case.